pkg/cmd/create/install_dependencies.go

```diff
@@ -37,19 +38,11 @@ var (
 		"kubectl",
 		"gcloud",
 		"helm",
-		"ibmcloud",
 		"tiller",
 		"helm3",
-		"kops",
-		"kvm",
-		"kvm2",
 		"ksync",
 		"oc",
```

Comment:
Ideally, we would not install this either imo. I think right now it is only used for some code for `jx install openshift`, but I doubt that is working. To keep things simple we might just keep it for now.